_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 ---
 layout: post
-title: "NewPipe 0.20.0 released: A new major version, a new unified player, a new notification experience... A lot is new this time!"
+title: "NewPipe 0.20.0 released: Major version, new unified player, new notifications. More things."
 short: "NewPipe 0.20.0 released"
 date: 2020-10-04 05:00:00 +0100
 categories: [pinned, release]
-author: Opusforlife & Poolitzer
+author: Opusforlife, Poolitzer and Allan Nordhøy
 image: newpipe
 excerpt_separator: <!-- more -->
 ---
 
-Hey there! Guess who is back? We are. Admittedly, we're quite a bit late with this release. But there is a _very_ good reason for that. Are you ready for it? Sit down, grab some popcorn and a beverage of your choice. This is going to be long.
+Hey there! Back admittedly a bit late with this release, but for _very_ good reason. Ready? Sit down, grab the virtual popcorn and some hydration.
 <!-- more -->
 
 ## A new unified player
 
-"What in the name of the coronavirus is the _unified player_?" you ask. Well, first of all, that's a very weird expression. Where did you even get that one from... - you know what? Save it. We don't want to know. You just keep on using that, we'll keep on talking about the new player.
+"If the new player is the _unified player_, was the old one un-unified?" you ask. The new player is new, and it is unified.
```

Comment:
Saying "the new player is new" doesn't really add anything.

Reply:
Therein lies the joke. Not sure it works.